modules/ossm-observability-access.adoc

```diff
@@ -4,39 +4,22 @@
 //* service_mesh/v2x/ossm-observability.adoc
 
 [id="ossm-observability-access-console_{context}"]
-= Accessing the Kiali console
+= Viewing service mesh data
 
-To access the Kiali console you must have {ProductName} installed and projects configured for the service mesh.
-
-.For administrators
-
-1. Use the perspective switcher to switch to the Administrator perspective. 
-
-2. Click *Home* > *Overview*.
+The Kiali operator works with the telemetry data gathered in {ProductName} to provide graphs and real time network diagrams of the applications, services, and workloads in your namespace.
 
-3. In the Launcher section, click `Open on Kiali`.
-
-4. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
-
-.For developers
+To access the Kiali console you must have {ProductName} installed and projects configured for the service mesh.
 
-1. Use the perspective switcher to switch to the Developer perspective. 
+.Procedure
 
-2. Click *Topology*.
+. Use the perspective switcher to switch to the Administrator perspective. 
 
-3. Click `Kiali` from a node in the topology you have configured for the service mesh. Or click the *Kiali* link in the toolbar.
+. Click *Home* > *Projects*.
 
-4. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+. Click the name of your project, for example `bookinfo`.
```

Comment:
nit: I'd make those two sentences with the second being "For example, click `bookinfo`."  The ISG topic "Inline examples" is loads of fun to read.  Use your judgement.